vhost-device-sound/Cargo.toml

```diff
@@ -38,3 +38,6 @@ rstest = "0.23.0"
 tempfile = "3.14"
 virtio-queue = { version = "0.14", features = ["test-utils"] }
 vm-memory = { version = "0.16.1", features = ["backend-mmap", "backend-atomic"] }
+
+[target.'cfg(target_env = "gnu")'.dev-dependencies]
```

Comment:
since the other dependencies are not target dependent, I would say avoid it or do it for everyone. Or did you have a specific problem?

Reply:
Only because it's used in the pipewire backend, and pipewire is only available under `"gnu"` environment. The other tests run with the `null` backend on non-gnu envs.